flocx_market/api/app.py

```diff
@@ -1,20 +1,17 @@
-import pecan
-from pecan import make_app
-from flocx_market import model
-from flocx_market.api import config as api_config
+import os
+from flask import Flask
+from flask import Blueprint
+from flask import jsonify
+my_bp = Blueprint("my_bp", __name__)
```

Comment:
I should have caught this before, but we probably want to rename this to something a bit less casual - flocx_market_api_bp or something